### PR TITLE
[#31044] Bottom footer style

### PR DESCRIPTION
**Pull request by HierenFeb**
https://agov_pr507.agov.com.au/styleguide/agov_whitlam/section-components.html#kssref-components-footer

### CODE REVIEW

File: themes/agov/agov_whitlam/sass/components/bottom-footer/_bottom-footer.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+//
+// Style guide: components.footer.bottom-footer
+
+.page-bottom-wrapper {
+  background-color: color(grey-extra-dark);
+  color: color(white);
+}
+
+.region-bottom {
+  @extend %layout-center;
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/bottom-footer/_bottom-footer.scss]: Don't need to extend %layout-center. Just use "layout-center in the markup.

File: themes/agov/agov_whitlam/sass/components/bottom-footer/_bottom-footer.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  a {
+    color: color(white);
+  }
+
+  &:after {
+    content: '';
+    clear: both;
+    display: table;
+  }
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/bottom-footer/_bottom-footer.scss]: Remove this and just use "clearfix" in the HTML markup.

File: themes/agov/agov_whitlam/sass/components/bottom-footer/_bottom-footer.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
+// Bottom Footer
+//
+// The bottom footer of page
+//
+// Markup: bottom-footer.hbs
+//
+// Style guide: components.footer.bottom-footer
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/bottom-footer/_bottom-footer.scss]: What's the reason for a 3 level nesting? I only see a single component here. `components.footer` should be sufficient.

File: themes/agov/agov_whitlam/templates/page.tpl.php [hunk 4]
```diff
@@ -113,4 +113,6 @@
 
 <?php print render($page['footer']); ?>
 
-<?php print render($page['bottom']); ?>
+<div class="page-bottom-wrapper">
+  <?php print render($page['bottom']); ?>
+</div>
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/templates/page.tpl.php]: This is the wrong region. "bottom" is a special region in page.tpl.php that isn't for blocks. You need to use the "footer" region.

Also, you don't need a wrapper div around the footer region in page.tpl. You can put all the classes you need in region--footer.tpl.php. You can also hard-code the "back to top" link in that region file.

File: themes/agov/agov_whitlam/sass/components/bottom-footer/_bottom-footer.scss [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    .copyright {
+      float: left;
+      margin: 0;
+  	}
+
+    .to-top {
+      float: right;
+    }
+  }
+}
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/bottom-footer/_bottom-footer.scss]: hmm? What's going on here? None of these rulesets are following a component architecture.

Use:
- `.footer`
- `.footer__link`
- `.footer__legal`
- `.footer__to-top`

**Comment by JohnAlbin** [comment]
Sorry. Meant footer-bottom, not footer.

**Comment by HierenFeb** [comment]
Wait a minute, I will add one more thing then you can review!

**Comment by HierenFeb** [comment]
@JohnAlbin: I updated! You can review it!

**Comment by kimpepper** [comment]
Feedback has been resolved.

**Pull request merged**